lib/rules/no-useless-escape.js

```diff
@@ -36,6 +36,8 @@ const VALID_STRING_ESCAPES = [
     "\r"
 ];
 
+const DIGITS = Array.from(Array(10).keys(), String);
```

Comment:
Yeah, maybe using Array#keys is overkill for 10 elements. I'll use that instead.